app/controllers/ajax/question_controller.rb

```diff
@@ -80,11 +83,10 @@ def create
         return
       end
 
-      # rubocop:disable Style/IfUnlessModifier
-      unless MuteRule.where(user: target_user).any? { |rule| rule.applies_to? question }
+      unless MuteRule.where(user: target_user).any? { |rule| rule.applies_to? question } ||
+          (author_is_anonymous && AnonymousBlock.where(identifier: AnonymousBlock.get_identifier(request.ip)).any?)
```

Comment:
won't this affect anon-blocks site-wide?  i.e., user A blocks anon X, which results in anon X not being able to ask user B anything

Reply:
I was about to come back here and actually just mention that.

Reply:
6ac8504